fvm/Cargo.toml

```diff
@@ -1,63 +1,54 @@
 [package]
 name = "fvm"
 description = "Filecoin Virtual Machine reference implementation"
-version = "4.0.0"
-license = "MIT OR Apache-2.0"
+version.workspace = true
+license.workspace = true
+edition.workspace = true
+repository.workspace = true
 authors = ["Protocol Labs", "Filecoin Core Devs"]
-edition = "2021"
-repository = "https://github.com/filecoin-project/ref-fvm"
 keywords = ["filecoin", "web3", "wasm"]
 
 [lib]
 crate-type = ["lib"]
 
 [dependencies]
-anyhow = { version = "1.0.71", features = ["backtrace"] }
-thiserror = "1.0.40"
-num-traits = "0.2"
+anyhow = { workspace = true, features = ["backtrace"] }
+thiserror = { workspace = true }
+num-traits = { workspace = true }
 cid = { workspace = true, features = ["serde-codec"] }
 multihash = { workspace = true, features = ["sha2", "sha3", "ripemd"] }
-fvm_shared = { version = "4.0.0", path = "../shared", features = ["crypto"] }
-fvm_ipld_hamt = { version = "0.9.0", path = "../ipld/hamt" }
-fvm_ipld_amt = { version = "0.6.2", path = "../ipld/amt" }
-fvm_ipld_blockstore = { version = "0.2.0", path = "../ipld/blockstore" }
-fvm_ipld_encoding = { version = "0.4.0", path = "../ipld/encoding" }
-serde = { version = "1.0", features = ["derive"] }
-serde_tuple = "0.5"
-lazy_static = "1.4.0"
+fvm_shared = { workspace = true, features = ["crypto"] }
+fvm_ipld_hamt = { workspace = true }
+fvm_ipld_amt = { workspace = true }
+fvm_ipld_blockstore = { workspace = true }
+fvm_ipld_encoding = { workspace = true }
+wasmtime = { workspace = true }
+wasmtime-environ = { workspace = true }
+wasmtime-runtime = { workspace = true }
+serde = { workspace = true }
+serde_tuple = { workspace = true }
+lazy_static = { workspace = true }
+log = { workspace = true }
+arbitrary = { workspace = true, optional = true, features = ["derive"] }
+rand = { workspace = true }
+quickcheck = { workspace = true, optional = true }
+once_cell = { workspace = true }
+minstant = { workspace = true }
+blake2b_simd = { workspace = true }
+byteorder = { workspace = true }
 derive_more = "0.99.17"
 replace_with = "0.1.7"
 filecoin-proofs-api = { version = "16", default-features = false }
 rayon = "1"
 num_cpus = "1.15.0"
-log = "0.4.19"
 fvm-wasm-instrument = "0.4.0"
 yastl = "0.1.2"
-arbitrary = { version = "1.3.0", optional = true, features = ["derive"] }
-rand = "0.8.5"
-quickcheck = { version = "1", optional = true }
-once_cell = "1.18"
-minstant = "0.1.2"
-blake2b_simd = "1.0.0"
-byteorder = "1.4.3"
 static_assertions = "1.1.0"
 ambassador = "0.3.5"
 
 [dev-dependencies]
 pretty_assertions = "1.3.0"
-fvm = { path = ".", features = ["testing"], default-features = false }
-
-[dependencies.wasmtime]
-version = "12.0.2"
-default-features = false
-features = ["cranelift", "pooling-allocator", "parallel-compilation"]
-
-[dependencies.wasmtime-environ]
-version = "12.0.1"
-
-[dependencies.wasmtime-runtime]
-version = "12.0.1"
-default-features = false
+fvm = { workspace = true, features = ["testing"], default-features = false }
```

Comment:
IIRC, `default-features = false` doesn't matter here (if you specify `workspace = true`). Honestly, I would just stick with the path dependency (I think we want that anyways).

Reply:
Maybe. I think cargo will warn if it's actually an issue?

Reply:
Hm. I guess this one works?